msteams-platform/apps-in-teams-meetings/in-meeting-notification-for-meeting.md

```diff
@@ -53,6 +53,99 @@ The following table provides the user types and lists the features that each use
+
+> [!NOTE]
+>
+> * Adaptive Cards aren't supported for targeted meeting notification.
+> * Targeted meeting notification is supported for scheduled meetings, instant meeting (Meet now), one-on-one calls, and group calls.
+
+In the following image, a meeting notification requesting payment is sent to one of the participants in the meeting. The meeting notification is only visible to the targeted participant:
+
+ :::image type="content" source="../assets/images/apps-in-meetings/reminder-sent.png" alt-text="Screenshot shows an example of a meeting notification sent to a targeted participant requesting for a payment. ":::
+
+### Enable app manifest settings for targeted meeting notification
+
+To send targeted meeting notifications, you must configure the `authorization` property and the `name` and `type` properties under the `resourceSpecific` field in the [app manifest](../resources/schema/manifest-schema.md#authorization) as follows:
+
+```json
+"authorization": {
+    "permissions": {
+      "orgWide": [],
+      "resourceSpecific": [
+        {
+          "name": "OnlineMeetingNotification.Send.Chat",
+          "type": "Application"
+        }
+      ]
+    }
+  }
+```
+
+### Enable targeted meeting notification
+
+> [!NOTE]
+> You can send a targeted meeting notification to 50 participants in a meeting. If you want to send a targeted meeting notification to more than 50 participants, you must call the `targetedMeetingNotification` API again.
+
+Targeted meeting notification can be triggered by user action.
+
+To enable the targeted meeting notification:
+
+1. Retrieve the user IDs of the participants through [Get participant API](meeting-apps-apis.md#get-participant-api) and [Get members API](../bots/how-to/get-teams-context.md#fetch-the-roster-or-user-profile).
+
+   **The following is an example of a user ID**:
+    `id=29:1I12M_iy2wTa97T6LbjTh4rJCWrtw2PZ3lxpD3yFv8j2YPnweY2lpCPPAn3RIOPP7rghfHauUz48I1t7ANhj4CA`
+
+1. Include the user IDs in the request parameter.
+
+   The following is an example of a request:
+
+    ```http
+    GET POST /v1/meetings/{meetingId}/notification
+    ```
+
+   The following is an example of a payload:
+
+    ```json
+    {
+      "type": "targetedMeetingNotification",
+      "value": {
+        "recipients": [ 
+        "29:1I12M_iy2wTa97T6LbjTh4rJCWrtw2PZ3lxpD3yFv8j2YPnweY2lpCPPAn3RI0PP7rghfHauUz48I1t7ANhj4CA"
+         ], 
+        "surfaces": [ 
+          { 
+            "surface": "meetingStage", 
+            "contentType": "task", 
+            "content": { 
+              "value": { 
+                "height": "300", 
+                "width": "400", 
+                "title": "Targeted meeting Notification", 
+                "url": "https://somevalidurl.com"           
+                }
+            } 
+          } 
+        ] 
+      },
+      "channelData": { // optional if a developer wants to support user attributes
```

Comment:
Shouldn't this be // optional if a developer does not want to support user attributes?

Reply:
We have defined the channel data property in meeting app APIs, Following is the screenshot for your quick reference:

<img width="463" alt="image" src="https://user-images.githubusercontent.com/91744210/213682731-7b02ae15-e370-4b94-9e41-c01afb3624e5.png">